build_compilers/lib/src/modules.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/element/element.dart';
+import 'package:build/build.dart';
+
+class Module {
+  /// The libraries in the strongly connected cycle.
+  ///
+  /// In most cases without cyclic imports this will contain a single asset.
+  final Set<AssetId> cycle;
```

Comment:
nit: I would just call this `srcs` so it is more general.

Reply:
Calling it `sources`